pkg/cmd/util/helpers.go

```diff
@@ -149,6 +151,33 @@ func RetrieveApplicationsByName(ctx context.Context, c client.Client, applicatio
 	return applicationMetaList, nil
 }
 
+func RetrieveApplicationStatusByName(ctx context.Context, c client.Client, applicationName string, namespace string) (ApplicationStatusMeta, error) {
+	var applicationStatusMeta ApplicationStatusMeta
+	var appConfig corev1alpha2.ApplicationConfiguration
+	if err := c.Get(ctx, client.ObjectKey{Name: applicationName, Namespace: namespace}, &appConfig); err != nil {
+		return applicationStatusMeta, err
+	}
+
+	component, err := GetComponent(ctx, c, appConfig.Spec.Components[0].ComponentName, namespace)
+	if err != nil {
+		return applicationStatusMeta, err
+	}
+
+	var workload corev1alpha2.WorkloadDefinition
+	err = json.Unmarshal(component.Spec.Workload.Raw, &workload)
+	if err != nil {
+		return applicationStatusMeta, err
+	}
+
+	traitDefinitionList := ListTraitDefinitionsByApplicationConfiguration(appConfig)
+	applicationStatusMeta = ApplicationStatusMeta{
+		Status:   string(appConfig.Status.Conditions[0].Status),
```

Comment:
same with above

Reply:
As for the `status` field, how can I get the status of the appConfig?  I also need to see more cases which have multiple `conditions` field as I couldn't figure out from the struct https://github.com/crossplane/crossplane-runtime/blob/master/apis/core/v1alpha1/condition.go#L76
```
status:
  conditions:
  - lastTransitionTime: "2020-08-04T13:04:24Z"
    message: 'cannot render components: cannot find trait definition "core.oam.dev/v1alpha2"
      "ManualScalerTrait" "manualscaler": TraitDefinition.core.oam.dev "manualscalertraits.core.oam.dev"
      not found'
    reason: Encountered an error during resource reconciliation
    status: "False"
    type: Synced
  dependency: {}
```